### ISSUE TITLE
Cant install module

### ISSUE THREAD

**Issue by noah3561** [TrueXPixels/quick.db#150]
When attempting to run `npm i quick.db`, i get this error

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! integer@2.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the integer@2.1.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-09-27T00_16_59_818Z-debug.log
```

**Notes:**

I am on the root account
I have already run `apt-get install build-essential`
I've already attempted to install integer manually via `npm i integer@2.1.0` to no avail.
I'm running on Ubuntu 18.04

**Comment by davidhaw** [comment]
What does the full log say that npm gives you in the last line of the error? I don't know if I'll be able to fully fix it but the log should give more information for anyone who wants to try to fix the error.

**Comment by ZoroSC** [comment]
Are you still having this issue?

**Comment by J0Miles** [comment]
I am having this issue as well. Within root account
I even tried `npm i node-gyp rebuild`. Problem still persists.

**Comment by ggtylerr** [comment]
Having similar issues on my repl (running on Linux):
```
--> npm install quick.db

> integer@2.1.0 install /home/runner/DSA-Bot-b/node_modules/integer
> node-gyp rebuild

make: Entering directory '/home/runner/DSA-Bot-b/node_modules/integer/build'
  CXX(target) Release/obj.target/integer/src/integer.o
  SOLINK_MODULE(target) Release/obj.target/integer.node
  COPY Release/integer.node
  COPY ../build/integer.node
  TOUCH Release/obj.target/place_resulting_binary.stamp
make: Leaving directory '/home/runner/DSA-Bot-b/node_modules/integer/build'

> better-sqlite3@5.4.3 install /home/runner/DSA-Bot-b/node_modules/better-sqlite3
> node-gyp rebuild

make: Entering directory '/home/runner/DSA-Bot-b/node_modules/better-sqlite3/build'
  TOUCH b857c92884e9598d609f6be182a2595df7a8e00f.intermediate
  ACTION deps_sqlite3_gyp_locate_sqlite3_target_extract_sqlite3 b857c92884e9598d609f6be182a2595df7a8e00f.intermediate
  TOUCH Release/obj.target/deps/locate_sqlite3.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite3/sqlite3.o
cc: internal compiler error: Killed (program cc1)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-6/README.Bugs> for instructions.
deps/sqlite3.target.mk:169: recipe for target 'Release/obj.target/sqlite3/gen/sqlite3/sqlite3.o' failed
make: *** [Release/obj.target/sqlite3/gen/sqlite3/sqlite3.o] Error 4
rm b857c92884e9598d609f6be182a2595df7a8e00f.intermediate
make: Leaving directory '/home/runner/DSA-Bot-b/node_modules/better-sqlite3/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:193:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.15.0-1036-gcp
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/runner/DSA-Bot-b/node_modules/better-sqlite3
gyp ERR! node -v v10.17.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok
npm WARN discord.js@11.5.1 requires a peer of @discordjs/uws@^10.149.0 but none is installed. You must install peer dependencies yourself.
npm WARN DSA-Bot-b@1.0.0 No description
npm WARN DSA-Bot-b@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! better-sqlite3@5.4.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the better-sqlite3@5.4.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-02-12T07_20_46_286Z-debug.log
exit status 1
```

From the looks of the log, it appears to be a better-sqlite3 issue. Actually looking into it reveals [this issue on their github.](https://github.com/JoshuaWise/better-sqlite3/issues/348) Basically, it's an npm issue and will likely be fixed on the next release. For now, just follow through the steps listed:
```
npm i -g node-gyp@latest
node-gyp -v
```
After that you should be able to install it normally.

**Comment by mokoshalb** [comment]
I believe my PR resolves this issue.

**Comment by TrueXPixels** [comment]
Most likely fixed with #160
